spec/models/order_spec.rb

```diff
@@ -59,15 +63,17 @@
     expect(Order.placed_for("an id")).to eq(nil)
   end
 
-  it "not find an order from previous week" do
-    order = {
-      user_name: "bob",
-      user_id: "an id",
-      lunch: "burger",
-      date: Days.monday - 1,
-    }
-    Order.place(order)
+  context "order from previous week" do
+    it "not find an order from previous week" do
+      order = {
```

Comment:
not sure about the value of this local variable here.

Reply:
I'll delete that